tests/example_experiments_test.py

```diff
@@ -0,0 +1,49 @@
+# pylint: disable=invalid-name,no-self-use
+import os
+import pyhocon
+
+from deep_qa.common.params import Params, replace_none
+from deep_qa.models import concrete_models
+
+from .common.test_case import DeepQaTestCase
+
+
+class TestExampleExperiments(DeepQaTestCase):
+    def setUp(self):
+        super(TestExampleExperiments, self).setUp()
+        self.example_experiments_dir = "./example_experiments"
+        self.entailment_dir = self.example_experiments_dir + "/entailment"
+        self.memory_networks_dir = self.example_experiments_dir + "/memory_networks"
+        self.multiple_choice_qa_dir = self.example_experiments_dir + "/multiple_choice_qa"
+        self.reading_comprehension_dir = self.example_experiments_dir + "/reading_comprehension"
+        self.sentence_selection_dir = self.example_experiments_dir + "/sentence_selection"
+        self.sequence_tagging_dir = self.example_experiments_dir + "/sequence_tagging"
+
+    def test_entailment_examples_can_be_parsed(self):
+        self.check_experiment_type_can_be_parsed(self.entailment_dir)
+
+    def test_memory_network_examples_can_be_parsed(self):
+        self.check_experiment_type_can_be_parsed(self.memory_networks_dir)
+
+    def test_multiple_choice_qa_examples_can_be_parsed(self):
+        self.check_experiment_type_can_be_parsed(self.multiple_choice_qa_dir)
+
+    def test_reading_comprehension_examples_can_be_parsed(self):
+        self.check_experiment_type_can_be_parsed(self.reading_comprehension_dir)
+
+    def test_sentence_selection_examples_can_be_parsed(self):
+        self.check_experiment_type_can_be_parsed(self.sentence_selection_dir)
+
+    def test_sequence_tagging_examples_can_be_parsed(self):
+        self.check_experiment_type_can_be_parsed(self.sequence_tagging_dir)
+
+    def check_experiment_type_can_be_parsed(self, directory):
+        for param_file in os.listdir(directory):
+            param_path = os.path.join(directory, param_file)
+            param_dict = pyhocon.ConfigFactory.parse_file(param_path)
+            params = Params(replace_none(param_dict))
+            model_class = concrete_models[params.pop("model_class")]
+            # Tests will try to create root directories as we have /net/efs paths,
+            # so we just remove the serialisation aspect here.
+            params["model_serialization_prefix"] = None
+            self.get_model(model_class, params)
```

Comment:
Maybe you if you also built the model here, you'd get more parameters read, and it would find the errors?  You'd have to do some mangling somewhere, though, because `_build_model()` will try to read the embedding file.  Probably you could check for `/net/efs/`, and replace it with `self.TEST_DIR`, then make sure that a toy embedding file has been written to the right place.